README.md

```diff
@@ -27,255 +20,154 @@ which functions as the data backbone for prominent hedge funds, banks, and finan
 | Java / Groovy | Yes                | Yes                |
 | C++           | No                 | Yes                |
 | JavaScript    | No                 | Yes                |
-| gRPC          | -                  | Yes                |
+| Go            | No                 | Yes                |
+| R             | No                 | Yes                |
+
+## Installation
+
+Deephaven Community Core can be used from both server- and client-side APIs. Server-side APIs allow users to connect directly to the Deephaven server and execute commands, write queries, and much more. Client-side APIs allow users to connect to a Deephaven server through a client application.
+
+See the following documentation links for installation instructions and more:
+
+- Python
+  - [Run from Docker](https://deephaven.io/core/docs/tutorials/quickstart/)
+  - [pip-installed](https://deephaven.io/core/docs/tutorials/quickstart/)
+- [Groovy]
+  - [Run from Docker](https://deephaven.io/core/groovy/docs/tutorials/quickstart/)
+- [Python client](https://pypi.org/project/pydeephaven/)
+- [Java client](https://deephaven.io/core/docs/how-to-guides/java-client/)
+- [JS client](https://deephaven.io/core/docs/reference/js-api/documentation/)
+- [Go client](https://pkg.go.dev/github.com/deephaven/deephaven-core/go)
+- [R client](https://github.com/deephaven/deephaven-core/blob/main/R/rdeephaven/README.md)
 
-## Run Deephaven
+Deephaven's client APIs use [gRPC](https://grpc.io/), [protobuf](https://github.com/deephaven/deephaven-core/tree/main/proto/proto-backplane-grpc/src/main/proto/deephaven/proto), [Apache Arrow Flight](https://arrow.apache.org/docs/format/Flight.html), and [Barrage](https://github.com/deephaven/barrage) to handle ticking data. Users who wish to build their own client APIs can use these tools to do so.
 
-This section is a quick start guide for running Deephaven from pre-built images.  Almost all users will want to run Deephaven using pre-built images.  It is the easiest way to deploy.  For detailed instructions, see [Launch Deephaven from pre-built images](https://deephaven.io/core/docs/tutorials/quickstart).
+## Build from source
```

Comment:
It seems like we've got a bit of duplication. I'm not sure if ideally we want this documentation to live in this readme here, or at https://deephaven.io/core/docs/how-to-guides/launch-build/, but it looks like the same content?

Reply:
I'm about to push a rework that gives a TL;DR for the three ways to run a Deephaven server, but with links to the appropriate documentation.